readthedocs/organizations/views/private.py

```diff
@@ -284,14 +286,15 @@ def get_queryset(self):
         )
         return self.filter.qs
 
-    def _get_retention_days_limit(self, organization):  # noqa
-        """From how many days we need to show data for this project?"""
-        return settings.RTD_AUDITLOGS_DEFAULT_RETENTION_DAYS
-
-    def _is_enabled(self, organization):  # noqa
-        """Should we show audit logs for this organization?"""
-        return True
-
+    def _get_retention_days_limit(self, organization):
+        """From how many days we need to show data for this organization?"""
+        return PlanFeature.objects.get_feature_value(
+            organization,
+            type=self.feature_type,
+        )
 
-class OrganizationSecurityLog(SettingsOverrideObject):
-    _default_class = OrganizationSecurityLogBase
+    def _is_feature_enabled(self, organization):
+        return PlanFeature.objects.has_feature(
+            organization,
+            type=self.feature_type,
+        )
```

Comment:
This is the same method as `_is_enabled`. However, it uses `organization=` instead of `project=` and it's named differently. We should decide whether naming it `_is_feature_enabled` or `_is_enabled` to keep consistency.